sample-plugin/pom.xml

```diff
@@ -343,17 +342,5 @@
                 <jenkins.version>2.222.4</jenkins.version>
             </properties>
         </profile>
-        <profile>
-            <id>skip-tests-on-release</id>
```

Comment:
No longer relevant due to `-Pquick-build` in https://github.com/jenkins-infra/jenkins-maven-cd-action/blob/b11e7369e037be500a81b3b02586b756c343ca4a/run.sh#L8.